mri/operators/fourier/cartesian.py

```diff
@@ -45,32 +46,42 @@ class FFT(OperatorBase):
         the mask samples in the Fourier domain.
     shape: tuple of int
         shape of the image (not necessarly a square matrix).
-     n_coils: int, default 1
-            Number of coils used to acquire the signal in case of multiarray
-            receiver coils acquisition. If n_coils > 1, data shape must be
-            [n_coils, Nx, Ny, NZ]
+    n_coils: int, default 1
+        Number of coils used to acquire the signal in case of multiarray
+        receiver coils acquisition. If n_coils > 1, data shape must be
+        [n_coils, Nx, Ny, NZ]
+    n_jobs: int, default 1
+        Number of parallel workers to use for fourier computation
     """
-    def __init__(self, samples, shape, n_coils=1):
+    def __init__(self, shape, n_coils=1, samples=None, mask=None, n_jobs=1):
         """ Initilize the 'FFT' class.
 
         Parameters
         ----------
-        samples: np.ndarray
-            the mask samples in the Fourier domain.
         shape: tuple of int
             shape of the image (not necessarly a square matrix).
-         n_coils: int, default 1
-                Number of coils used to acquire the signal in case of
-                multiarray receiver coils acquisition. If n_coils > 1,
-                 data shape must be equal to [n_coils, Nx, Ny, NZ]
+        n_coils: int, default 1
+            Number of coils used to acquire the signal in case of
+            multiarray receiver coils acquisition. If n_coils > 1,
+            data shape must be equal to [n_coils, Nx, Ny, NZ]
+        samples: np.ndarray, default None
+            the mask samples in the Fourier domain.
+        mask: np.ndarray, default None
+            the mask as a matrix with 1 at sample locations
+            please pass samples or mask
         """
-        self.samples = samples
```

Comment:
One test is failing because this is missing, can you check it out?

Reply:
Yes, need to rebase i guess

Reply:
cool, let's see how the tests do